python/ray/autoscaler/_private/monitor.py

```diff
@@ -168,10 +166,7 @@ def __init__(self,
         head_node_ip = redis_address.split(":")[0]
         self.redis_address = redis_address
         self.redis_password = redis_password
-        if os.environ.get("RAY_FAKE_CLUSTER"):
-            self.load_metrics = LoadMetrics(local_ip=FAKE_HEAD_NODE_ID)
-        else:
-            self.load_metrics = LoadMetrics(local_ip=head_node_ip)
+        self.load_metrics = LoadMetrics()
```

Comment:
RAY_FAKE_CLUSER seems useful. Not needed any more?

Reply:
local_ip is no longer an initialization parameter for LoadMetrics